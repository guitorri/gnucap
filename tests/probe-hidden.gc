"""
spice
.print tran hidden(0)
.tran 0 2 1 trace=all
.tran 0 2 1 trace=all
.tran 0 2 1
.tran 0 2 1 skip=10
.tran 0 2 1 skip=10 trace=all
.end
"""
