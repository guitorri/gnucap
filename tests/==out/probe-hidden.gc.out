#Time       hidden(0) 
 0.         0.        
 0.01       0.        
 1.         0.        
 2.         0.        
#Time       hidden(0) 
 0.         0.        
 0.01       0.        
 1.         0.        
 2.         0.        
#Time       hidden(0) 
 0.         0.        
 1.         1.        
 2.         0.        
#Time       hidden(0) 
 0.         0.        
 1.         10.       
 2.         10.       
#Time       hidden(0) 
 0.         0.        
 0.001      0.        
 0.1009     0.        
 0.2008     0.        
 0.3007     0.        
 0.4006     0.        
 0.5005     0.        
 0.6004     0.        
 0.7003     0.        
 0.8002     0.        
 0.9001     0.        
 1.         0.        
 1.0999     0.        
 1.1998     0.        
 1.2997     0.        
 1.3996     0.        
 1.4995     0.        
 1.5994     0.        
 1.6993     0.        
 1.7992     0.        
 1.8991     0.        
 1.999      0.        
 2.         0.        
